### PR TITLE
Carving docs update

### PR DESCRIPTION
**Pull request by m-novikov**

### CODE REVIEW

File: documentation/carving/carving.md [hunk 1]
```diff
@@ -128,48 +127,53 @@ These additional options described below can be displayed by scrolling down
...
+<a href="screenshots/carving-04-labeling-options.png" data-toggle="lightbox"><img src="screenshots/carving-04-labeling-options.png" class="img-responsive" /></a>
 
 
 ## Saving and loading segmented objects
 
-Once the user has successfully segmented an object, the segmentation result
+Once the user has successfully segmented an object, the result
 can be stored by clicking on the **Save As** button. A dialog will pop up that
-asks for the objects name:
+asks for the object name, by default autogenerated name will be provided based on value of **Object name prefix** field:
```
Comment (k-dominik) [inline documentation/carving/carving.md]: Maybe just mention that this autogeneration appends numbers to the object prefix

Comment (m-novikov) [reply documentation/carving/carving.md]: reworded a bit

File: documentation/carving/carving.md [hunk 2]
```diff
@@ -109,6 +111,7 @@ Additional available interactions include:
 
 - **Updating the segmentation:** Left click on button **Segment**
 - **Erasing a brush stroke:** Change brush mode to eraser by clicking on the eraser button
+- **Window Leveling** Change brush mode by clicking on window leveling tool button <img src="/documentation/basics/screenshots/window_leveling.png" />
```
Comment (k-dominik) [inline documentation/carving/carving.md]: Have you maybe forgotten to commit the image?

File: documentation/carving/carving.md [hunk 3]
```diff
@@ -128,48 +131,53 @@ These additional options described below can be displayed by scrolling down
...
 Sometimes it is necessary to refine the segmentation of an already saved object. To do so, click
 on the **Browse objects** button and load the corresponding object. 
 
-<a href="screenshots/carving-7.png" data-toggle="lightbox"><img src="screenshots/carving-7.png" class="img-responsive" /></a>
+<a href="screenshots/carving-07-browse.png" data-toggle="lightbox"><img src="screenshots/carving-07-browse.png" class="img-responsive" /></a>
+
+Right-clicking on the highlighted object in **Completed segments** overlay and navigating to a submenu with object name (in this case it's "Object 1")
+provides the following options:
+
+- Load _ObjectName_
```
Comment (k-dominik) [inline documentation/carving/carving.md]:
```suggestion
- Load _object name_ 
```
(just for consistence ith the text before)

**Pull request merged**